apps/ps.go

```diff
@@ -19,15 +20,31 @@ func Ps(app string) error {
 	t := tablewriter.NewWriter(os.Stdout)
 	t.SetHeader([]string{"Name", "Amount", "Size", "Command"})
 
+	hasAutoscaler := false
 	for _, ct := range processes {
+		name := ct.Name
+		_, err = autoscalers.GetFromContainerType(app, name)
```

Comment:
You need a cache here, you can't make a list request at each iteration of the loop

(same at the other place of the PR where you're doing this)

Reply:
@EtienneM You get all the autoscalers with a call in #index once, then you look into them instead of making plenty of API requests.

Reply:
Silly me... Sorry. That's obvious to me now